package.json

```diff
@@ -65,6 +65,7 @@
     "hls.js": "^0.13.1",
     "howler": "^2.1.3",
     "intersection-observer": "^0.7.0",
+    "jellyfin-apiclient": "https://github.com/MrTimscampi/jellyfin-apiclient-javascript#jf-web-part2",
```

Comment:
I guess we want to change that to our repo?

Reply:
This is in progress, I'm waiting on a tag for 1.1.0 to pin this against the org's repository.